src/bin/cql-stress-cassandra-stress/settings/command/user.rs

```diff
@@ -59,6 +60,22 @@ pub struct QueryDefinition {
     pub serial_consistency: Option<SerialConsistency>,
 }
 
+impl QueryDefinition {
+    pub async fn to_prepared_statement(&self, session: &Arc<Session>) -> Result<PreparedStatement> {
+        let mut statement = session
+            .prepare(&*self.cql)
+            .await
+            .with_context(|| format!("Failed to prepare statement: {}", self.cql))?;
+
+        if let Some(consistency) = self.consistency {
+            statement.set_consistency(consistency);
+        }
+        statement.set_serial_consistency(self.serial_consistency);
```

Comment:
I'm not sure if this is what we want about `serial_consistency`. If `QueryDefinition` contains `None::<Consistency>`, then I assume that the intended semantics is that we just want the default consistency. I assume this is the semantics of `None::<SerialConsistency>` as well. The way you did it, however, makes `None::<SerialConsistency>` override the default `Some(SerialConsistency::LocalSerial)` with `None`.

Reply:
Good catch, thanks!